### ISSUE TITLE
Don't say "Patches were not touched. All were applied properly" when there are no patches

### ISSUE THREAD

**Issue by scop** [phracek/rebase-helper#206]
For example, https://bugzilla.redhat.com/show_bug.cgi?id=1351714#c3

> Patches were not touched. All were applied properly

The package in question does not have any patches in the first place, so the message is at best misleading. It would be good to omit the message altogether, and thus cause less noise in Bugzilla and email.

**Comment by phracek** [comment]
@scop Yes, this is definitely weird. We plan to introduce new version of rebase-helper which will have better format for representing data like JSON.
